k8s/netmanager-app/templates/hpa.yaml

```
@@ -0,0 +1,29 @@
+  name: '{{ .Values.app.name }}-hpa'
+  namespace: {{ .Values.app.namespace }}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ .Values.app.name }}
+  minReplicas: {{ .Values.autoscaling.minReplicas }}
+  maxReplicas: {{ .Values.autoscaling.maxReplicas }}
+  metrics:
+    {{- if .Values.autoscaling.targetCPUUtilizationPercentage }}
+    - type: Resource
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)